static/app/components/u2f/u2finterface.tsx

```diff
@@ -111,18 +129,52 @@ class U2fInterface extends React.Component<Props, State> {
       });
   }
 
+  webAuthnSignIn(authenticateRequests) {
+    const credentials = [] as any;
```

Comment:
Can this be typed better? what is `PublicKeyCredentialRequestOptions['allowCredentials']`?

Reply:
this is a  dictionary of the Web Authentication API from https://developer.mozilla.org/en-US/docs/Web/API/PublicKeyCredentialRequestOptions, i can leave a comment about this

Reply:
Can't you set the types more appropriately though?

Reply:
`PublicKeyCredentialDescriptor[]` no?

Reply:
ah i see what you mean